WebAppServiceBus/WebAppServiceBus/ReceivedMessageStore.cs

```diff
@@ -9,30 +8,18 @@ namespace WebAppServiceBus
 {
     public static class ReceivedMessageStore
     {
-        private static QueueClient _receiveClient = null;
         private static List<string> _receivedMessages = new List<string>();
 
-        public static void Initialize(ServiceBusConfiguration config)
+        public static async void InitializeMessage(ServiceBusConfiguration config)
         {
-            if (_receiveClient != null)
-            {
-                return;
-            }
+            ServiceBusReceiver receiver = new ServiceBusClient(config.NamespaceConnectionString).CreateReceiver(config.Queue);
 
-            TokenProvider tokenProvider = TokenProvider.CreateManagedServiceIdentityTokenProvider();
+            ServiceBusReceivedMessage receivedMessage = await receiver.ReceiveMessageAsync();
 
-            _receiveClient = new QueueClient($"sb://{config.Namespace}.servicebus.windows.net/", config.Queue, tokenProvider, receiveMode: ReceiveMode.ReceiveAndDelete);
```

Comment:
This should be converted to using the ServiceBusProcessor rather than the receiver - https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/servicebus/Azure.Messaging.ServiceBus#using-the-processor